tests/App/AppKernel.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\App;
+
+use Symfony\Component\Yaml\Yaml;
+use FOS\RestBundle\FOSRestBundle;
+use Symfony\Component\HttpKernel\Kernel;
+use Sonata\MediaBundle\SonataMediaBundle;
+use Symfony\Bundle\TwigBundle\TwigBundle;
+use JMS\SerializerBundle\JMSSerializerBundle;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
+use Doctrine\Bundle\DoctrineBundle\DoctrineBundle;
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\Routing\RouteCollectionBuilder;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class AppKernel extends Kernel
+{
+    use MicroKernelTrait;
+
+    public function __construct()
+    {
+        parent::__construct('test', false);
+    }
+
+    public function registerBundles()
+    {
+        return [
+            new FrameworkBundle(),
+            new SecurityBundle(),
+            new TwigBundle(),
+            new FOSRestBundle(),
+            new SonataMediaBundle(),
+            new JMSSerializerBundle(),
+            new DoctrineBundle()
+        ];
+    }
+
+    public function getCacheDir(): string
+    {
+        return $this->getBaseDir().'cache';
+    }
+
+    public function getLogDir(): string
+    {
+        return $this->getBaseDir().'log';
+    }
+
+    public function getProjectDir(): string
+    {
+        return __DIR__;
+    }
+
+    protected function configureRoutes(RouteCollectionBuilder $routes)
+    {
+        $routes->import(__DIR__.'/routes.yml', '/', 'yaml');
+    }
+
+    protected function configureContainer(ContainerBuilder $containerBuilder, LoaderInterface $loader)
+    {
+        $loader->load(__DIR__.'/config.yml');
+        $config = Yaml::parse(file_get_contents(__DIR__.'/security.yml'));
```

Comment:
```suggestion
        $loader->load(__DIR__.'/security.yml');
```
I think you can do the same: